### PR TITLE
Show bitcoinD/stratisD command line arguments

### PR DESCRIPTION
**Pull request by Neurosploit**

### CODE REVIEW

File: Stratis.Bitcoin/Configuration/NodeSettings.cs [hunk 1]
```diff
@@ -362,5 +362,34 @@ private static string GetDefaultDataDir(string appName, Network network)
...
+			builder.AppendLine("-conf=<Path>			Path to the configuration file.");
+			builder.AppendLine("-datadir=<Path>			Path to the data directory.");
+			builder.AppendLine("-testnet=<0 or 1>		Use the testnet chain.");
+			builder.AppendLine("-regtest=<0 or 1>		Use the regtestnet chain.");
+			builder.AppendLine("-acceptnonstdtxn=<0 or 1>	Accept non-standard transactions. Default 1.");
+			builder.AppendLine("-maxtipage=<number>		Max tip age. Default " + DEFAULT_MAX_TIP_AGE.ToString());
+			builder.AppendLine("-server=<0 or 1>		Accept command line and JSON-RPC commands. Default 0.");
+			builder.AppendLine("-rpcuser=<string>		Username for JSON-RPC connections");
+			builder.AppendLine("-rpcpassword=<string>		Password for JSON-RPC connections");
+			builder.AppendLine("-rpcport=<0-65535>		Listen for JSON-RPC connections on <port>. Default: 8332 or (reg)testnet: 18332");
```
Comment (dangershony) [inline Stratis.Bitcoin/Configuration/NodeSettings.cs]: Can need to take the default form the settings as well

File: Stratis.BitcoinD/Program.cs [hunk 2]
```diff
@@ -16,28 +16,35 @@ public class Program
 	{
 		public static void Main(string[] args)
 		{
-			var loggerFactory = Logs.GetLoggerFactory(args);
-			Logs.Configure(loggerFactory);
-			
-			NodeSettings nodeSettings = NodeSettings.FromArguments(args);
+			if (args != null && args.Length == 1 && (args[0].StartsWith("-help") || args[0].StartsWith("--help")))
```
Comment (dangershony) [inline Stratis.BitcoinD/Program.cs]: Lets try to keep the program.cs as clean as possible so maybe something like

```
if(NodeSettings.PrintHelp(loggerFactory , args))
     return
```

**Comment by dangershony** [comment]
Nice work mate, just let me know when you work on a task so there wont be duplicate work by mistake

File: Stratis.BitcoinD/Program.cs (hunk 2 above)
Comment (Neurosploit) [inline Stratis.BitcoinD/Program.cs]: Sure.

**Comment by dangershony** [comment]
Excellent, couldn't have done it better myself.

**Pull request merged**